test/util.ts

```diff
@@ -142,9 +73,9 @@ export abstract class TestBuilder {
 
     // TODO: Use testModule in these cases?
     protected tsHeader = "";
-    public setTsHeader(tsHeader: string): this {
+    public setTsHeader(tsHeader: string | undefined): this {
```

Comment:
Why is this | undefined here? I don't think it should be, better to put the `?? ""` at the call site